server/image_remove.go

```diff
@@ -22,9 +22,23 @@ func (s *Server) RemoveImage(ctx context.Context, req *types.RemoveImageRequest)
 	return s.removeImage(ctx, imageRef)
 }
 
+func (s *Server) untagImageForAllImageServers(img string) error {
+	iServer, err := s.StorageImageServerPerImage(img)
+	if err != nil {
+		return err
+	}
+	for _, is := range iServer {
```

Comment:
in many cases, there is a specific multiStore endpoint for this kind of batch iterator work, why not one here?